start_cerberus.py

```diff
@@ -441,15 +441,13 @@ def main(cfg):
 
                 # Sleep for the specified duration
                 logging.info("Sleeping for the specified duration: %s\n" % (sleep_time))
-                time.sleep(float(sleep_time))
-
                 sleep_tracker_start_time = time.time()
+                time.sleep(float(sleep_time))
 
                 # Track pod crashes/restarts during the sleep interval in all namespaces parallely
                 multiprocessed_output = pool.starmap(
-                    kubecli.namespace_sleep_tracker, zip(watch_namespaces, repeat(pods_tracker))
+                    kubecli.namespace_sleep_tracker, zip(watch_namespaces, pods_tracker)
```

Comment:
@chaitanyaenr I think the main problem is here that since we aren't setting repeat to "n" number of times it endlessly polls pods_tracker which is not correct. This needs to be taken out 
```
class repeat(object):
    """
    repeat(object [,times]) -> create an iterator which returns the object
    for the specified number of times.  If not specified, returns the object
    endlessly.
    """
```